### PR TITLE
Cleanup Docs

### PR DESCRIPTION
**Pull request by hakman**
This fixes all warnings regarding outdated links and updates the main page.

### CODE REVIEW

**Review by johngmyers — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -11,18 +11,14 @@
...
-
 ## What is kOps?
 
 We like to think of it as `kubectl` for clusters.
 
 `kops` will not only help you create, destroy, upgrade and maintain production-grade, highly
 available, Kubernetes cluster, but it will also provision the necessary cloud infrastructure.
 
-AWS (Amazon Web Services) is currently officially supported, with DigitalOcean, GCE, and OpenStack in beta support, and Azure and AliCloud in alpha.
+[AWS](getting_started/aws.md) (Amazon Web Services) is currently officially supported, with [DigitalOcean](getting_started/digitalocean.md), [GCE](getting_started/gce.md), and [OpenStack](getting_started/openstack.md) in beta support, and [Azure](getting_started/azure.md) and AliCloud in alpha.
```
Comment (johngmyers) [inline docs/index.md:21]: Strange, GCE is feature flagged and marked as alpha in the code.

File: docs/releases/1.4-NOTES.md [hunk 2]
```diff
@@ -12,8 +12,7 @@ Reworked SSH keys and support for running CI builds
   do a rolling update when you would otherwise not have to.  I suggest waiting till you next upgrade kubernetes.
 
 * Builds that are not published as Docker images can be run.  `kops` will pass a list of images in the NodeUp
-  configuration, and NodeUp will download and `docker load` these images.  For examples, see the
-  [testing tips](../development/testing.md)
+  configuration, and NodeUp will download and `docker load` these images.
```
Comment (johngmyers) [inline docs/releases/1.4-NOTES.md:15]:
```suggestion
  configuration and NodeUp will download and `docker load` these images.
```

**Review by johngmyers — APPROVED** [review]
/hold
in case you want to fix the nit.

File: docs/index.md (hunk 1 above)
Comment (hakman) [reply docs/index.md:21]: I think it's something that we should fix and help it graduate to stable. Will add on office hours list of topics.

**Pull request merged**